src/lib/tokenPrices.ts

```diff
@@ -73,6 +73,7 @@ const getLootPrice = async () => {
 	);
 
 	const ethToUsd = await tokenToUsd('ethereum');
+	if (!ethToUsd) return;
```

Comment:
You've changed the return type of this function from `Promise<number>` to `Promise<number | undefined>` which isn't okay. if `tokenToUsd` fails, it should throw an exception, not 0 or undefined.